code/modules/awaymissions/corpse.dm

```diff
@@ -40,6 +40,30 @@
 	M.timeofdeath = timeofdeath
 	if(src.mob_species)
 		M.set_species(src.mob_species)
+	if(ishuman(M))
+//		random_character()
+//		M.prefs.random_character()
+		var/mob/living/carbon/human/H = M

```

Comment:
uh....this already all typecasted to a human mob, no need to retypecast it as a human mob, again.